app/javascript/react/components/BrandShowContainer.js

```diff
@@ -39,12 +39,12 @@ const BrandShowContainer = (props) => {
     })
     
     return(
-    <div>
+    <>
```

Comment:
this block should be indented under the return, which would also correct the indentation on lines 48 & 49